app/controllers/api/auth/stories_controller.rb

```diff
@@ -1,14 +1,13 @@
 # encoding: utf-8
 
 class Api::Auth::StoriesController < Api::StoriesController
-
   include ApiAuthenticated
 
   api_versions :v1
 
   filter_resources_by :account_id, :series_id, :network_id
 
-  filter_params :noseries
+  filter_params :highlighted, :purchased, :v4, :text, :noseries
```

Comment:
Also, it's slightly strange to define the `filter_params` here, when it's the filtered `super(resources)` call that actually applies them.